lua/autorun/server/sv_restrict_propcore.lua

```diff
@@ -41,6 +47,26 @@ function restrictPropCoreFunctions()
             end
         end
     end
+    
+    for _, signature in pairs( globalrestrictedFunctions ) do
+        if wire_expression2_funcs then
+            local oldFunc = wire_expression2_funcs[signature][3]
+
+            wire_expression2_funcs[signature][3] = function( self, ... )
+                if ( self.player:IsAdminI() ) then
+                    local isInBuildMode = self.player:GetNWBool("CFC_PvP_Mode", false) == false
+
+                    if( isInBuildMode or self.player:IsAdmin() ) then
```

Comment:
Verification for IsAdmin isn,t needed as it is checked above
`if isInBuildMode then`

Reply:
Fixed! I hope I haven't done anything else wrong.